K2 Cloud regional endpoints (#97)
C2Devel/terraform-provider-croccloud: website/docs/index.md

  required_providers {
    aws = {
      source  = "c2devel/rockitcloud"
      version = "~> 25.2"
    }
  }
}

# Configure the rockitcloud provider.
# The section is named `aws` for backward compatibility.
provider "aws" {

>   # For K2 Cloud: specify one of the supported regions.

Reviewer: можно и так, но нейтрально -- просто зпт вместо двоеточия (тогда ниже тоже надо поменять)

Author: использовала запятые

>   # For other cloud platforms: specify a non-empty string,

Reviewer: любую строку можно указать? тогда просто enter, так как specify подразумевает, что мы что-то определенное должны ввести

Author: Да, любую. Поменяла на enter

The Rockit Cloud Provider manages resources via API calls. To connect the provider to the cloud platform,
you must specify the platform API endpoints in the `provider.endpoints` block or define the corresponding environment variables.

-> **Note**

> When connecting to K2 Cloud, you can specify one of [the supported regions][k2-supported-regions] in the configuration.

Reviewer: вводная фраза не избыточна в контексте?
```suggestion
You can specify one of [the supported regions][k2-supported-regions] in the configuration.
```

Author: Думаю, что комментарий в конфигурации легко упустить из виду, и лучше продублировать эту информацию в тексте.

> In this case the `provider.endpoints` block can be omitted,

Reviewer: ```suggestion
In this case, the `provider.endpoints` block can be omitted,
```

> because all API endpoints will be generated by the provider based on the region value.

Reviewer: желательно активный залог использовать (не догма)
```suggestion
because the provider will generate all API endpoints based on the region value.
```
